### PR TITLE
Add Translations

### PR DESCRIPTION
**Pull request by GoneTone**
Add Translations (undone)

### CODE REVIEW

**Comment by PEMapModder** [comment]
Keep both authors, separated with a comma >.<

**Comment by GoneTone** [comment]
OK

**Comment by GoneTone** [comment]
對 json 不太熟 :p

**Comment by GoneTone** [comment]
這樣嗎?

**Comment by PEMapModder** [comment]
yep :smile: and thanks

**Comment by PEMapModder** [comment]
I won't merge this until you have completed translations for all phrases whose value `updated` is `1.5` or less have been translated.

Also, please update `meta.native.updated` to `1.5`, as `meta.comment` indicated that `meta.native.updated` should represent the latest English translation version this translation is updated for..

**Comment by PEMapModder** [comment]
Please also help me rename it to `zh-hant` instead of `zh-tr`. I just realized that this is not the best way of systematic naming of language files. I will write an article about systematic naming later.

**Comment by GoneTone** [comment]
OK

**Comment by PEMapModder** [comment]
I think we will use this, but all in lowercase.
http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry

**Comment by GoneTone** [comment]
嗯

**Comment by GoneTone** [comment]
完成翻譯了，但是部份我不確定這樣翻譯對不對

File: resources/lang/zh-hant.json [hunk 1]
```diff
@@ -0,0 +1,221 @@
+{
+	"meta": {
+		"language": {
+			"value": "zh-hant",
+			"updated": "1.5"
```
Comment (PEMapModder) [inline resources/lang/zh-hant.json]: Please change this line into `1.1`.

File: resources/lang/zh-hant.json [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+		},
+		"authors": {
+			"value": [
+				"PEMapModder",
+				"旋風之音 (GoneTone)"
+			],
+			"updated": "1.0"
+		},
+		"comment": {
+			"value": "主要的版本會在每一次新的插件釋放時會有衝突本次的版本應該會修正此提交(包括合併提交)的en.json文件會被更新。所有未使用的翻譯作品仍必須有`更新`的權力版本被定義在meta.language.updated，有關英文的版本則被定義在meta.native.updated",
```
Comment (PEMapModder) [inline resources/lang/zh-hant.json:19]: You actually don't need to translate this. These 4 lines should be deleted :P But whatever, it is OK.

**Comment by PEMapModder** [comment]
Never mind, I will merge it and update them myself.

**Comment by GoneTone** [comment]
OK

**Comment by Tolo0** [comment]
Are you bringing a updated version of WorldEditArt out with copy and paste?

**Comment by PEMapModder** [comment]
@Tolo0 I will.

**Comment by Tolo0** [comment]
I can't wait. It's already the best plugin ever. And with copy and paste. It would just be more amazing

**Comment by EnderKenji** [comment]
Wht about zh_Hans?

**Comment by PEMapModder** [comment]
@EnderKenji translate it yourself.

**Pull request merged**